tests/RulesetTest.php

```diff
@@ -148,7 +151,7 @@ public function testSerializing($record)
 
         try {
             if ($record->header_type == 'item') {
-                $serializedValue = Serializer::serializeItem($record->expected[0], $record->expected[1]);
+                $serializedValue = Serializer::serializeItem($record->expected);
             } else {
                 $serializedValue = Serializer::{'serialize' . ucfirst($record->header_type)}($record->expected);
             }
```

Comment:
Conditional is no longer needed with updates to `convertExpectedItem` to return an `Item` object.